docs/organizations/overview.mdx

```diff
@@ -5,26 +5,141 @@ description: Learn about how to create and manage Clerk organizations and their
+Organizations play a crucial role in structuring user management within your application, particularly in B2B, B2B2C, and collaborative environments. Think of a platform like GitHub Teams, or even departments within a company - organizations are a way to group and manage users and their access to resources.
 
-There is no limit to the number of organizations a user can be a member of. However, a user can only create up to 100 organizations in a given application instance.
+By utilizing organizations, you can assign specific roles and permissions to users, allowing you to finely control access to different parts of your application. Whether you are managing projects, coordinating teams, or facilitating partnerships, organizations offer a flexible and scalable way to structure user access and permissions.
+
+<Callout type="info">
+  To explore organizations in Clerk, check out this demo repo:
+  https://github.com/clerk/organizations-demo
+</Callout>
 
 ## Enable organizations in your application
 
+Organizations are disabled by default.
+
 To enable organizations:
-1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=organizations-settings) and select your application.
+
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=organizations-settings).
 2. In the navigation sidebar, select **Organizations Settings**.
 3. Toggle on **Enable Organizations**.
 
-## Implement organization features in your application
+Once organizations are enabled, you will be presented with the default settings, roles, and permissions that are applied to all organizations in that application instance. The following sections will explain these settings in more detail.
 
-Clerk provides [prebuilt components](/docs/components/overview), [React hooks](/docs/references/react/use-organization), and [APIs](/docs/references/javascript/organization/organization) to help you integrate organizations into your application. Using the [Organizations Backend API](https://clerk.com/docs/reference/backend-api/tag/Organizations), you can provide additional [private and public metadata for the organization](/docs/organizations/metadata) and set custom attributes.
+### Roles and permissions
 
-<Callout type="info">
-  To explore Clerk's Organizations, check out this demo repo:
-  https://github.com/clerk/organizations-demo
-</Callout>
+Roles determine a user's level of access and permissions within an organization. [Learn more about how roles and permissions work and how to create your own with Clerk.](/docs/organizations/roles-permissions)
```

Comment:
```suggestion
Roles determine a user's level of access and permissions within an organization. [Learn more about how roles and permissions work and how to create your own with Clerk](/docs/organizations/roles-permissions).
```

IDK why but most people prefer the period outside the link, and I think in most of the docs we do that, so 🤷 let's just do it here

Reply:
that's so funny you caught this too ! I sat doing back and forth and looking at it in preview, and I was like idk which looks better 🤔 imo, I think it's better on the inside because when it's rendered, the text is highlight purple but the period is highlighted the text colour (white) and it looks funny.

Reply:
That's fine with me 👍 Let's leave it inside